contracts/seed/SeedFactory.sol

```diff
@@ -80,7 +77,7 @@ contract SeedFactory is CloneFactory, Ownable {
         bool _isWhitelisted,
         uint8 _fee,
         bytes32 _metadata
-    ) public onlyOwner returns (address) {
+    ) public onlyOwner isInitialized returns (address) {
```

Comment:
2) I will have to pass argument here too. Which causes ```stack too deep``` error.

Reply:
> 1. I will have to pass argument here too. Which causes `stack too deep` error.

Why do you have to pass an argument?

Reply:
I guess I don't understand your point